db/projects/nano/project.yaml

```diff
@@ -0,0 +1,146 @@
+name: Nano
+web:
+  - 'https://nano.org'
+start_date: '2016-02-29'
+description: "Nano is a low-latency payment platform that requires minimal resources; making Nano ideal for peer-to-peer transactions."
+ledgers:
+  - id: nano
+    name: Nano
+    type: blockchain
+    networks:
+      - id: main
+        type: main
+        name: Block lattice
+        proof_type: DPoS
+        mineable: false
+        algorithm: DAG
+
```

Comment:
Tree said that the empty lines does not matter, but anyway, I would deleted them